<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterConfig</class>
 <widget class="QWidget" name="FilterConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Source Code Pro&quot;;
color: #8fbed4;
background-color: rgb(31, 38, 38);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DeadZone</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_deadzone_sld">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rot_deadzone_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>WNoise</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_smooth_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TNoise</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_smooth_sld">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Accela_Check">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Accela Filter</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="rot_input_max_label_yaw">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>InputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_output_max_label_yaw">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_output_max_sld_yaw">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="rot_input_max_sld_yaw">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Translation</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_smooth_label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TNoise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>InputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_input_max_sld_z">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DeadZone</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_deadzone_label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>WNoise</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_output_max_sld_z">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="trans_deadzone_sld">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="trans_smooth_sld">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="trans_input_max_label_z">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_output_max_label_z">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_input_max_label_pitch">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw:InputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>InputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_output_max_label_pitch">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw:InputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_input_max_sld_pitch">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_output_max_sld_pitch">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_input_max_sld_roll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="rot_output_max_sld_roll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rot_output_max_label_roll">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw:InputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_input_max_label_roll">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw:InputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>InputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_expo_label_yaw">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expo</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_expo_sld_yaw">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rot_expo_label_pitch">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_expo_sld_pitch">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expo</string>
   </property>
  </widget>
  <widget class="QLabel" name="rot_expo_label_roll">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expo</string>
   </property>
  </widget>
  <widget class="QSlider" name="rot_expo_sld_roll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Yaw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Pitch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_expo_label_z">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_expo_sld_z">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_input_max_label_y">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>InputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_input_max_sld_y">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="trans_output_max_sld_y">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_expo_label_y">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_expo_sld_y">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="trans_output_max_label_y">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="trans_expo_label_x">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_input_max_sld_x">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="trans_output_max_label_x">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>InputMax</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_output_max_sld_x">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expo</string>
   </property>
  </widget>
  <widget class="QSlider" name="trans_expo_sld_x">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>410</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="trans_input_max_label_x">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothing</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Accela_Check_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Double Accela Filter</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
